backend/src/models/Company.ts

```diff
@@ -0,0 +1,28 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const companySchema = new Schema({
+  _id: {
+    type: String,
+    unique: true,
+    required: true,
+  },
```

Comment:
MongoDB will automatically populate `_id` for models, so this field can be removed. The reason it is included for the User model is because we are customizing the ID using the google authentication id. Apart from this everything looks good!

Reply:
And just FYI the automatic `_id` field will have a type of `ObjectId`.

Reply:
Heard that!